### ISSUE TITLE
fix or deactivate npm warnings

### ISSUE THREAD

**Issue by funewik** [arfedulov/semantic-ui-calendar-react#91]
**Describe the bug**
Hello,

Is there any way to fix or deactivate following error messages :

```
npm WARN semantic-ui-calendar-react@0.14.0 requires a peer of semantic-ui-css@^2.4.1 but none is installed. You must install peer dependencies yourself.
npm WARN semantic-ui-calendar-react@0.14.0 requires a peer of semantic-ui-react@^0.84.0 but none is installed. You must install peer dependencies yourself.
```
It works well with `semantic-ui-less` (no need to `semantic-ui-css`) and `semantic-ui-react` version 0.85.0.

**Dependencies versions**

* semantic-ui-react : 0.85.0
* semantic-ui-less : 2.4.1
* semantic-ui-calendar-react : 0.14.0

Thanks

**Comment by arfedulov** [comment]
fixed in 0.14.1

**Comment by funewik** [comment]
Thanks !
